### ISSUE TITLE
post workshop version

### ISSUE THREAD

**Issue by sythel** [dams-mcda/Dams-MCDA#177]
If we are going to progress the application further we will need to decide the fate of PR #174 
because merging would remove needed content

the biggest problem going forward still seems to be #151

**Comment by sythel** [comment]
I propose close #174 and tag it with version v1.5.0

**Comment by elbfox** [comment]
Yes, versioning is what I was going to suggest here, too. Do you mind waiting until I run this by Sharon in our meeting tomorrow? Is this something we can have an archived link for, much like http://shiny.gsscdev.com/dams_mcda/ ?

**Comment by sythel** [comment]
we could make another "archive" site or we could update the archive site, given there's no use for version v1.0.0 anymore.

**Comment by elbfox** [comment]
Making a new archive site is preferred because I'm interested in tracking the development of the tool in concert with participant feedback over time. It would be best to be able to reference the specific version and link to it where possible.

**Comment by sythel** [comment]
v1.0.0 server is shiny.gsscdev.com
v1.5.0 server is shiny2.gsscdev.com
latest version will stay dams-mcda.gsscdev.com
launching master as latest

**Comment by sythel** [comment]
I want to give a heads up; I don't know how many hours I have left that I can contribute to this project.

**Comment by elbfox** [comment]
> I want to give a heads up; I don't know how many hours I have left that I can contribute to this project.

Can you get a sense from Mike how many hours you do have? This will help us prioritize the remaining tasks and make a final call about which version of the model to use.

**Comment by sythel** [comment]
> Can you get a sense from Mike how many hours you do have? 

As soon as I find out I'll let you know.

**Comment by sythel** [comment]
> > Can you get a sense from Mike how many hours you do have?
> 
> As soon as I find out I'll let you know.

Around 30 hours, some of that has to include re-launching DDC which will take a few hours

**Comment by sythel** [comment]
@elbfox I am halting working on this today unless you let me know of something to work on. I could attempt #78

**Comment by elbfox** [comment]
That is not a lot of time...yikes! It may be that we need to do our improvements on the non-MOGA version, then, if there is not enough time left to integrate it and finish troubleshooting.

Knowing this, priority issues are: #180, #149 , #146 , and I have a few more to add today. I will prioritize them with tags. If you want to spend time working on these and let me know when you're done, I'll review changes ASAP.

**Comment by elbfox** [comment]
Can the non-MOGA version (http://shiny2.gsscdev.com, for example) become the "main" version, at this point in time? I don't know what this means for switching branches around, or time spent, but we don't have time to troubleshoot and get the multi-dam model (with the NormalizedMatrix dataset and maps) working before your hours are up.  

This may change if we can dig up additional funds and you have time/interest in continuing to work on this with us, but we should definitely merge new changes with the non-MOGA branches. I find GitHub challenging to keep track of, so let me know if this is NOT something you are already doing.

**Comment by sythel** [comment]
> I don't know what this means for switching branches around, or time spent,

The only thing that takes a decent amount of time is making another server instance altogether, in comparison to just updating an existing server instance with changes.

If we want we could move branch MCDA_Only_No_MOGA to master, and make sure to tag the old master as v2.0.0 so we can find it easier at a later point in time.

**Comment by sythel** [comment]
out of things to work on

**Comment by elbfox** [comment]
> If we want we could move branch MCDA_Only_No_MOGA to master, and make sure to tag the old master as v2.0.0 so we can find it easier at a later point in time.

This sounds like a good plan.  

> out of things to work on
 
Can you pause work on this then, until I can submit a PR? I am still working on everything I need to include there.

**Comment by sythel** [comment]
[v2.0.0](https://github.com/dams-mcda/Dams-MCDA/tree/v2.0.0)
after PR #187 master will be v1.5.0

**Comment by sythel** [comment]
need input on something to work on

**Comment by elbfox** [comment]
I think we need to pause work right now until we get funding sorted.

**Comment by sythel** [comment]
@elbfox Did you happen to fill out the DDC portal content form?

**Comment by elbfox** [comment]
Yes, some time ago. Brianna had requested it before the summer. I will need to update the metadata/data dictionary information and project description now, but I was hoping to be able to meet with Sharon again, first (this will be in November).  We have some potential changes to make (which I will be responsible for when I get back from Iceland) requiring updates to some of our documents included in text-based links and revisions in criteria descriptions.

**Comment by sythel** [comment]
can you forward it to my email? I can't seem to find it. I just want to create the empty pages, I will wait to populate content.

**Comment by sythel** [comment]
we use the format ddc-<project>.sr.unh.edu for the website names

is ddc-dam-toolbox.sr.unh.edu an acceptable domain name for the portal? or would you like something different

**Comment by elbfox** [comment]
Can we call it ddc-dam-decision-support-tool.sr.unh.edu? 

@sharonklein Billy is building our portal (this was necessarily included in his remaining hours) so if you have objections to this please let us know

**Comment by sythel** [comment]
> Can we call it ddc-dam-decision-support-tool.sr.unh.edu?

If you would like. It isn't a huge deal except for cases when your typing in the address manually vs. going to DDC and finding it under the projects section.

**Comment by elbfox** [comment]
The one website name word I would cut is "tool" but that strikes me as the thing that @sharonklein would want to keep. I think it is still not very long, so I'd say let's just keep it. I rarely ever enter websites manually...usually copy-paste or go through a portal/link

**Comment by sharonklein** [comment]
website name looks good to me and building portal sounds good. I can't make
any move on funding until I hear from Mike and Carol. So, I am stuck
waiting until then. With Emma leaving on vacation tomorrow, we may have to
get what we can done until then and then revisit how to proceed when she
returns the week of Nov 4 - hopefully by then we will know one way or
another if we have enough funds to proceed beyond what we have at that
point.

Thanks for all of your hard work on this - both of you!

Sharon Klein, Ph.D.
Associate Professor
School of Economics
5782 Winslow Hall, Room 305
University of Maine
Orono, ME 04469
207-581-3174
http://umaine.edu/soe/faculty-and-staff/klein/

**Comment by sythel** [comment]
@elbfox 
Let me know if there are issues I can assist with. I am testing the interface for the new introduction modals in the meantime.

**Comment by elbfox** [comment]
Thank you, I think the modals and the citations list are the only remaining things I need help on. I have one final PR to do, and I'm almost done getting those changes queued up for a push to the DDST_v4 branch (again, I've messed this up but don't have the mental bandwidth right now to figure out starting a new branch unless my mistake prevents me from merging with the master branch), and we should be all set. Emailing you and Michael shortly with a 'packet' of final information from Sharon and me.

**Comment by elbfox** [comment]
Can you let me know when the final version weblink is ready @sythel? it looks like the dev version (shiny2.gsscdev.com) is down right now

**Comment by sythel** [comment]
> 
> 
> Can you let me know when the final version weblink is ready @sythel? it looks like the dev version (shiny2.gsscdev.com) is down right now
> 
> ![image](https://user-images.githubusercontent.com/19410104/83771040-d5558400-a64f-11ea-9d86-3c405f1facbc.png)

I think UNH or the server room lost power

**Comment by sythel** [comment]
version 2.1.1 includes label fix #211 

https://github.com/dams-mcda/Dams-MCDA/tree/v2.1.1
